docs/2_3_common_states.adoc

```diff
@@ -346,7 +346,7 @@ If `stopTimeDefined == fmi3False`, then no final value of the <<independent>> va
 
 Function `fmi3Set{VariableType}`::
 
-This function can be called for variables with <<variability>> latexmath:[\neq] <<constant>> and with <<initial>> = <<exact>> or <<approx>>.
+This function can be called for variables with <<variability>> latexmath:[\neq] <<constant>> and with <<initial>> = <<exact>> or <<approx>>, or variables with <<causality>> = <<input>>.
```

Comment:
Wouldn't it make sense to require `inputs` to be `initial = exact`?

Reply:
in a way yes, but exact input implies that they are known before hand and this is not true in many cases, so approx. would make more sense as setting this before enter initialization mode is just a start value for eventual iteration unless the input is unconnected. However approx. also forbids you to change them during initialization wich does not apply to inputs.. 

In general i think they were exact, in some beta release of fmi 2 this was removed and this bug that you wasn't allow to set them was introduced.

I would be fine to return the exact on inputs as they behave in the same way as variables with initial exact during initialization.

Reply:
Currently we forbid variables of causality `input `to have an `initial` attribute.  
@t-sommer : do you propose to change this?

Reply:
I think it would be cleaner to require `initial = exact` for inputs since we could get rid of two exceptions in the text. Also IMHO it does not matter if the value is a guess value. It will always be exactly this value.